### DISCUSSION TITLE
Server routes 404 in Cloudflare Pages

### DISCUSSION THREAD

**Discussion by jasonhibbs** [nuxt/nuxt#24692] [Questions]
I'm evaluating hosting and Nuxt says Cloudflare Pages is a zero config solution, but all my /api routes 404.

https://wurdit.pages.dev/

Is this expected and intended for static sites only?

I'm currently running the same deploy fine on Netlify at https://wurd.it

**Comment by rayblair06** [comment]
Did you ever find a solution to this? We have a similar problem with SSG & i18n returning 404 on routes that should be redirected to the correct page.

**Reply by jasonhibbs** [reply]
No – others say they have no problems, but I never found the time to investigate a fresh project to compare
